assets/icons/file_icons/smithy.svg

```diff
@@ -0,0 +1,9 @@
jyvJEc8TmL9uxsOEuFLoEhde7IK0ZTK5KpWkPuMkQaUx8t+7bDk26FcbkPTzqqr4g14rNoI+1DWZ3ia3Tnnk3633Arx5E0Cp79uUaUdfja84LsaAHqUafSGz2bYB3TsoemiApKwGsqL70NSLQJNYoWSa3SKvN3iQOSEgXAXR3XsYQmiS9gwhi034w7jsQuOibA62FN6Dz6S/CroLMCyRO1oP7axEu2Dx1F0lBCqFPt0hHV83Yvreju6d5nkIqfIU9YxihbAswy0IRfRtjSo5Qos3PLar/W7BQCE1JlBQh24Y4fqtSPzKPa5w/+Mkx7cV5Jm0Sh1Tw3JKGfRKT5oNFhzdQ8F6M4u/RNHJHKGQIWZMxCaVS8ktIeCcMe9Ufo6VPXTZI/q1Zh1lC0hABnLApZZhhtHD2Kh6vEO+EE2TAr14i7A9ZxdWVXeT84BnCf7HFQdLoCkKxlVq/prSir7JMUawrzdYkDpkQN7Zi3ZaicAeG7y8DE0LOZxgvniWGYNxh9F4rr80eqnWooroT1Zde14oWnOmaxLoJESgJY4xm0WwNhDBqKcwtqjRsAbRj0lUXcmbZBKrHIynf9so/XU3CpqQc0zabpZ8SFiH+sQjft5NwTlehKhTVl05SeDPiP3FCPrbODK0vnYUMH+xgJGz+/Jvjc0YNrUfL48cIPwp/Oj4gmoQ+TlaimtQ+LTmt/o0jR/R2cIPJcthhdBQw7DRDFrAT1VeSJG9BzZiNyHrz3Iy497lcKZth6HWNDISccR0RQmr26pBvSJTJ7+/5inMNczVkK/70fuEDEfcZi+XknbE8n2IKQgSrtpKGry2U3AMFwVSz7vn/QWi3L29utt4bq6ToVX+BUVQcmts/UlW+BU3tUMfV/p9fDLMwThd3uJKfj7XqyzQa4kYza3vNfsbYbXiu0K0pnKRqhD+emNB2mzhGyi07Fq6mI0SAlv1e9ZcJVKzQ71wpo9emDEIfZWV7mvy7WKq+TFdleX7FdTMzL3I66Vr0M2bgva6yINL3iLo0sUlZGwudR12F8AQl2veOGTkXEkUphE0RI9B6y9PZJE5qUl+ONimmrLI8P4Lcdyu/gk25Ga2nHXh/xupLaE8wTjSJl3WksdujvUJS7xcVTCF7NIxjUsZILkmrMdg5E9qi60P7ofriS50u6/pojRL3GkIE++X2zIsoY69gIiCclTJiQYaDcLbYVlJT3aNfFRLrVYR0ktK5JJa8hnsx1RxG+UTnk27D/42Kyuvzy+oOQViQNaDIiT4XRob1JdgTsWowyeXiZB0KNw0I6qq+PPJ5HPflONfxbafCtomZTQ8/w297JSECJbF4nDEimsRiSWzY5YQcrCfjH+OMsIezSurEFsCIaEvYGTX8EzFQYOXU7FFcUwshMgPoGVJWyPmMcnpbpOxLuOpsIHzGixJHx2LBxS0Ytt+F1tcZTWI9KYLVizHf/4Jj2lgxFWC469WECLTEHD/X+DzsWRQrJI2qZi7Hb5psqrJn2o1mxBA1NjpTEZBHq6eMG4XT+jejwDmGyaekEV2eX+UVVdcYJbOvENKJ1y4caSsRXowqZ6hRAKI6rNW4cueEkoaPjJDZ66ssT5AmlFR/AjKW4p1RVRdaCnws1qK95JiWacj28j5FiCCHtUiv4PKhuDfMUZqJJsNzjsnhk9LnCOlcHU9xXCGheudRvPKIRsPV6Dw+W1mQcanXAEG+7HOECFzAxG7UWnVBYhR8MEqyNS6td0zPGRl8pO4h+yQh1qPqftiSt2BI2rvDYcjTJK6oL++ePG6MHml9khAxCcWo/CYKv18PaIHj8FwcIrOuFCMFgcN2D9EnCREQiF48fqzyL7iNhJaIgRpbAldXVNhtw7pD7v+pzxIiYHE5ratByko0gk/4h0mXL36Ci/yME+ez4vdVgpXQpzqG3kDZUTDGauXJC7F14X6A0e13db2F1/EOpoqWqJTPm1BU+0Wg+H1aQwQ4U4s+bMVe9+UYqn8ej5gtNNyBZ27HHspn66fmnRdIep/XEDdA9bMyUtra2VNYKX8X3uFgA8Od6Pe8qsmu+8Zv33PEl/Q4IR7IiKM6eKr2ELbDLcDrCJFC32KJbH7OVu8zj32+yvLgg4he/PcSwf5GsUWbdHj6GXSP/Th8ttqhrd09/aqzvcmMa4gXVBrz0/s1ySmrcOrQ7fCOxNpfBcMsmxSn8tDEsobvPLMQ1xBPNLru08saW2hbwkIsJ8Ip36TNS5BwX8loE8+xWNiScpstzVNYXEM80TjtvvPo9WTXH9FXwd58GomzVDBnRl5nkjQfHdUmkXxcQ04jwfPR5nCcIG2WpSDjr3hv6OhwVzoW2KsbVU19qhTVpHgX15AuZPxdRLVCk5xrgNZNCBcJm9KOs8PWJCUqS2H14y4QApu+/rr91mFnl0mylIqwV+JPDhQnRH8h72pNkb6La0gIyGGd1yCuakthU+YimtGkiJx8GrchIRAiDG8b54/CpryJaIbbFMzPH4xrSAiEuIOKTh1TLCvwjG11xtgUEPEZI+yOuIa4UQ7hKsaiGJMWYIR4I6KFPSApyOBEW9R1ilIIOYkH7YZAuW10Gk1OWovGqtAUfR83pccQ9xe2oppiIVyfEBEz7khexUfHKLUsxBcuziEO3aZQ8jnT+E1uMgSkcULC/LDEkVRtGr0bg4ai8xj8GSqc7AGRc3NKand4ZiFOiCcaOu/tO2sOyRp7EB1HnBkZxCp7Tj6ljN97YMCFu05PEiTFnVEIiB/ktFjEzCOdBZm+sD1MuHodzm6p8JZuXEO8oaLz3cSyum8TKLkbA4Zvowo7fT+KWE9cj2H3Gb7IEMnGCdEJvq9oY4vq9smczkPn8TUPUvBDQbSOSmR+XlGN39/y9aVWvtKLvw8SgZrJYwe7KF+EQcMbMa/SoFLX4vHFH+wF4EJTfLr/AeGisZCpHNYMAAAAAElFTkSuQmCC" x="-50" y="-50" width="100" height="100"></image>
```

Comment:
This doesn't look like a true SVG, as it is embedding a PNG inside of it.

We need the icons to in SVG format so that they scale properly without artifacting.

Also, once this is addressed, can we also make the icon just in black and white?